SingularityBase/src/main/java/com/hubspot/singularity/DeployStyle.java

```diff
@@ -0,0 +1,5 @@
+package com.hubspot.singularity;
+
+public enum DeployStyle {
```

Comment:
I don't want to overcomplicate things, but I wonder about the merits of an enum vs. being able to define a batch size for the deploy that defaults to the total instance count (i.e. a "normal" deploy)